draft-ietf-httpbis-digest-headers.md

```diff
@@ -233,7 +237,7 @@ whereas "digest" is only used in relation to the value contained in the fields.
 
 Integrity fields: collective term for `Content-Digest` and `Repr-Digest`
 
-Integrity preference fields: collective term for `Want-Repr-Digest` and `Want-Content-Digest`
+Integrity preference fields: collective term for `Want-Repr-Digest` and `Want-Content-Digest`.
```

Comment:
This change is unrelated. But it also is inconsistent with the line above it. So something needs to budge.

The RFC editor will decide what they want, so maybe just drop this change for now.